content/docs/pipelines/caching.md

```diff
@@ -0,0 +1,110 @@
++++
+title = "Caching"
+description = "Getting started with Kubeflow Pipelines step caching"
+weight = 50
++++
+
+Starting from Kubeflow Pipelines 0.4, Kubeflow Pipelines supports step caching capabilities in both standalone deployment and GCP hosted deployment.
+
+## Before you start
+
+This guide tells you the basic concepts of Kubeflow Pipelines step caching and how to use it. 
+This guide assumes that you already have Kubeflow Pipelines installed or want to use standalone or GCP hosted deployment options in the [Kubeflow Pipelines deployment 
+guide](/docs/pipelines/installation/) to deploy Kubeflow Pipelines.
+
+## What is step caching?
+
+Kubeflow Pipelines caching provides step-level output caching. 
+And caching is enabled by default for all pipelines submitted through the KFP backend and UI. 
+The exception is pipelines authored using TFX SDK which has its own caching mechanism. 
```

Comment:
Is there documentation for the TFX Pipeline caching mechanism? If so, please add a link to it.

Reply:
Currently not. Thanks!